### PR TITLE
Issue 350

### LINKED ISSUE #350: Create upstream pre-processed authoritative source
Revert vrs.yaml to be a yaml representation of vrs.json, and add upstream authoritative source for y2j.py

### PR DESCRIPTION
**Pull request by ahwagner**
closes #350. Also demonstrates the utility of the new tooling; accidentally put `id` as required in an earlier commit; correcting to `_id` automatically updates corresponding information models and JSON Schema!

### CODE REVIEW

**Comment by ahwagner** [comment]
Closed & reopened this PR to kick off RTD preview auto-build.

File: schema/Makefile [hunk 1]
```diff
@@ -5,10 +5,13 @@
 YAMLS:=vrs.yaml
 JSONS:=${YAMLS:.yaml=.json}
 
-all: ${JSONS} defs
+all: vrs.json defs
```
Comment (reece) [inline schema/Makefile:8]: What about ga4gh.json?

File: schema/defs/Variation.rst [hunk 2]
```diff
@@ -16,9 +16,9 @@ A representation of the state of one or more biomolecules.
       - Description
    *  - _id
       - :ref:`CURIE`
-      - 0..1
+      - 1..1
```
Comment (reece) [inline schema/defs/Variation.rst:19]: `_id` is optional. Should be `0..1`, right?

**Review by reece — APPROVED** [review]
Looks good. Thanks for the quick action!

**Comment by larrybabb** [comment]
> Looks good. Thanks for the quick action!

That's what they said?

**Review by larrybabb — APPROVED** [review]

File: schema/Makefile (hunk 1 above)
Comment (ahwagner) [reply schema/Makefile:8]: AFAIK, ga4gh.json isn't used anywhere, and unlike vrs.json, isn't JSON Schema.

It's not a big deal to put it back in, but is it actually our intent to provide this file for some purpose, or is the YAML sufficient?

File: schema/defs/Variation.rst (hunk 2 above)
Comment (ahwagner) [reply schema/defs/Variation.rst:19]: Oh wow. Thanks @reece, great catch. Will fix.

File: schema/Makefile (hunk 1 above)
Comment (reece) [reply schema/Makefile:8]: I provided for completeness (json and yaml pairs), but your logic is sound and that file should probably move to ga4gh core anyway. So, let's keep as-proposed and we'll revive if needed.

**Pull request merged**